packages/bson-transpilers/test/yaml/partial.yaml

```diff
@@ -49,6 +49,7 @@ tests:
           csharp: ''
           shell: NumberInt
           ruby: ''
+          swift: Int32
     - input:
```

Comment:
So the way the templates for these work is you can define two different templates: one that handles the type name itself and one that handles the arguments. The transpiler then automatically puts these together to get the full construction (e.g. `"Int32(5)"` = `"Int32"` + `"(5)"`).

Only `Int32` uses the template for the type name though, since it's the only one that has a consistent initialization invocation. Longs, for example, use the integer literals sometimes. decimal128 needs to have `try` in front. For these ones, we leave the type name template blank and include it as part of handling the args instead (e.g. `""` + `"try BSONDecimal128(\"5\")"` = `"try BSONDecimal128(\"5\")"`).

For the ones that don't use the type name template, we can just omit the test case or leave the expectation blank. The other languages do this too for the ones they skip.

Reply:
ahhh I see. that makes sense. thanks for the explanation!